examples/java7/build-java7.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -e
+
+which acbuild || (echo "Make sure acbuild is on your path"; exit 1)
+
+TMPDIR=$(mktemp -d)
+PACKAGESDIR=$(mktemp -d)
+
+rmDirectories() {
+    rm -rf ${TMPDIR}
+    rm -rf ${PACKAGESDIR}
+}
+trap rmDirectories EXIT
+
+pushd ${PACKAGESDIR}
+apt-get download \
+    ca-certificates-java \
+    java-common \
+    libasound2 \
+    libatk1.0-0 \
+    libatk-wrapper-java-jni \
+    libc6 \
+    libcairo2 \
+    libcups2 \
+    libfontconfig1 \
+    libfreetype6 \
+    libgcc1 \
+    libgdk-pixbuf2.0-0 \
+    libgif4 \
+    libglib2.0-0 \
+    libgtk2.0-0 \
+    libkrb5-3 \
+    liblcms2-2 \
+    libnss3 \
+    libpango-1.0-0 \
+    libpangocairo-1.0-0 \
+    libpangoft2-1.0-0 \
+    libpcsclite1 \
+    libpng12-0 \
+    libpulse0 \
+    libstdc++6  \
+    libx11-6 \
+    libxext6 \
+    libxi6 \
+    libxinerama1 \
+    libxrandr2 \
+    libxrender1 \
+    libxtst6 \
+    openjdk-7-jre \
+    openjdk-7-jre-headless \
+    tzdata-java \
+    zlib1g \
+    bash \
+    libncurses5 \
+    libtinfo5
+popd
+
+shopt -s nullglob
+for d in ${PACKAGESDIR}/*.deb; do
+  echo "Extracting $d"
+  dpkg-deb -x ${d} ${TMPDIR}
```

Comment:
Doing that - You are skipping postinst process, with also skip of post install triggers (ldconfig for example).
Maybe using debootstrap with `--include` parameter would be more secure option for doing above work?